ndm-operator.yaml

```diff
@@ -142,6 +142,8 @@ spec:
         - name: procmount
           mountPath: /host/proc
           readOnly: true
+        - name: devmount
```

Comment:
yes, 
For handling cases where the device is with the pool is connected after the NDM pod has started. i.e the pool is created on a device which was claimed using the legacy UUID. The following sequence of events will happen
1. NDM 0.6 is used and user creates a pool on `/dev/sdb` (blockdevice-abc which uses legacy UUID)
2. User disconnects the `/dev/sdb`
3. Upgrade to the new version of NDM which support partitions
4. Connects the drive, assume it came back at `/dev/sdb`. In this case we should be able to read the filesystem on the newly connected disk. Then only we can understand that the device is currently in use. 
Otherwise NDM will try to create resources for the partitions `sdb1` and `sdb9`  which should not happen.

If we are able to access and read the disk, then we can identify that disk is already in use. 

@pawanpraka1 You see any issues with mounting this. Similar to the bug that was hit with open FDs.

Reply:
To make it clear, can we add comments to each of the mount points indicating why we need them.

Reply:
we should have done this long back. A big thumbs for this change.